fbrp/src/fbrp/runtime/docker.py

```diff
@@ -171,6 +171,18 @@ def _build(self, name: str, proc_def: ProcDef, args: argparse.Namespace):
                 elif "errorDetail" in lineinfo:
                     util.fail(json.dumps(lineinfo["errorDetail"], indent=2))
 
```

Comment:
Maybe you want to "always pull" when the tag contains "latest" so we always run on the new version.
Also it would be logically helpful to put the code in the else section of the L152 `if self.dockerfile`  to highlight we do not need to pull for dockerfile specified container.

Reply:
Done